### PR TITLE
add groupable field

### PR DESCRIPTION
**Pull request by rleiwang-traceableai**
With groupable field, UI will render attribute in group-by widget iff groupable == true

### CODE REVIEW

**Comment by aaron-steinfeld** [comment]
How does this get defaulted?

**Comment by rleiwang-traceableai** [comment]
the default value is false, at https://github.com/hypertrace/attribute-service/pull/5/files#diff-ea085817933ca044a95fd50b543cb420R51

**Comment by aaron-steinfeld** [comment]
> the default value is false, at https://github.com/hypertrace/attribute-service/pull/5/files#diff-ea085817933ca044a95fd50b543cb420R51
1. Please remove the comments about the UI - they're irrelevant to attribute service. This is a general setting that can be used throughout the platform (we should enforce this in query at some point later)
2. I disagree with the default - we should maintain the current behavior, which is that any string typed attribute is by default groupable. Config should have to opt out, rather than opt in.

File: attribute-service-impl/src/main/java/org/hypertrace/core/attribute/service/model/AttributeMetadataModel.java [hunk 1]
```diff
@@ -47,8 +47,7 @@
   @JsonProperty(value = "tenant_id")
   private String tenantId;
 
-  /** if true, UI will render the attribute in group-by widget */
-  private boolean groupable;
+  private boolean groupable = true;
```
Comment (aaron-steinfeld) [inline attribute-service-impl/src/main/java/org/hypertrace/core/attribute/service/model/AttributeMetadataModel.java:50]: Not all attributes are groupable though - we just want strings right now. I don't think we get away from logic here. How do the other fields get their defaults set? The default aggregations for example are set based on the attribute type.

**Pull request merged**